plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Bookmarking.js

```diff
@@ -126,10 +126,7 @@ Ext.define('NX.controller.Bookmarking', {
   onNavigate: function (token) {
     var me = this;
 
-    if (token !== Ext.History.bookmark) {
```

Comment:
so this looks like it was here to prevent self-navigation - is that no longer an issue? Seems unrelated to the FF bug